CHANGELOG.rst

```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
+    node:del       node.del
+    prop:set       node.prop.set
+    prop:del       node.prop.del
+    tag:add        node.tag.add
+    tag:del        node.tag.del
+    ============== ==============
 
-- Cortex map_async is now default
+Why make the change
+    By placing all node edit permissions under a single hierarchy, administrators may easily grant access to all node modifications by adding the permission ``node``.  As previously, more granular versions of the given permissions are available to individually control access.
 
-- Lots of remote API changes
+What you need to do
+    The Synpase ``0.2.x`` migration tool should take care of translating all relevant permissions changes within a migrated Cortex.  However, any 3rd party code that modifies permissions should be updated to the new naming convention.
 
-- Old clients in synapse.tools can't talk to new (>=0.2.0) cortex.  New clients in synapse.tools can't talk to < 0.2.0 cortex.
+Node Edits vs Splices
+~~~~~~~~~~~~~~~~~~~~~
 
-- Remote layer removed
+What changed
+    The various splice events provided by the storm runtime have been consolidated and streamlined into a single ``node:edits`` event type.  The new ``node:edits`` events contain a more compressed and aggregated representation of node changes and include changes that were previous unreported.  Callers may now specify the format for the node edits reported by the storm runtime by using the ``editformat`` optional parameter to the storm runtime, but the default output will now be ``node:edits`` rather than splices.  The following table enumerates the options for the ``editformat`` parameter:
 
-- Deprecation of lots of methods
+    ========== =========================================================
+    editformat Storm runtime behavior
+    ========== =========================================================
+    nodeedits  Provide new ``node:edits`` events.
+    splices    Provide backward compatible splice events.
+    count      Provide ``node:edits:count`` events with simple counts.
+    none       Do not transmit any representation for node edits.
+    ========== =========================================================
 
-- Removed cortex offset storage
+Why make the change
+    The splice format was originally designed to facilitate a single atomic edit per-splice.  As such, it required the potentially large primary property value to be embedded in each splice.  When making multiple edits, this representation is inefficient and causes the retransmission, and potential storage, of duplicate data.  Additionally, the key-value structure of the splice format provided unnecessary extensibility at the cost of transmission/storage size.
+
+What you need to do
+    Update any code that consumes/indexes the various splice events to handle the new ``node:edits`` format.  Additionally, callers may specify ``editformat: "splices"`` within their storm runtime options to enable backward compatible splice generation.
```

Comment:
backward-compatible

Reply:
Not according to merriam-webster and most of the internet...

Reply:
done